packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js

```diff
@@ -86,6 +97,15 @@ export default class QuanticNumericFacet extends LightningElement {
     item.end
   )}`;
   /**
+   * The sort criterion to apply to the returned facet values. Possible values are:
+   *   - `ascending`
+   *   - `descending`
+   * @api
+   * @type {'integer' | 'decimal'}
+   * @defaultValue `'integer'`
+   */
+  @api withInput;
```

Comment:
no default value is actually set here. default value should be described as undefined and i think it might be good to explain in the description that no input is the default behaviour.